starcounter-include.html

```diff
@@ -462,6 +463,7 @@
          * @TODO: to be simplified after https://github.com/StarcounterApps/CompositionProvider/pull/13 is done
          */
         StarcounterIncludePrototype.saveLayout = function(compositionStr){
+			console.warn("This method of saving composition is deprecated and will be removed soon.");
```

Comment:
I think this warning is useless without linking to or explaining an alternative. If there isn't one yet, I'd just remove it

Reply:
I updated the comment and @alshakero agreed with my correction. Merging.